README.md

```diff
@@ -192,3 +196,48 @@ query Terms {
   }
 }
 ```
+
+### Look up terms by URI
+
+Use the `lookup` query to look up terms whose URIs you know (for example, because you have stored the URIs previously):
+
+```graphql
+query {
+  lookup(
+    uris: ["https://data.rkd.nl/artists/32439", "https://data.cultureelerfgoed.nl/term/id/cht/15e29ea3-1b4b-4fb2-b970-a0c485330384"],
+  ) {
+    uri
+    source {
+      ... on Source {
+        uri
+        name
+        creators {
+          uri
+          name
+          alternateName
+        }
+      }
+      ... on Error {
+        __typename
```

Comment:
`__typename` is not present on line 238. Should we omit it here, or add it on line 238? Or should `__typename` be placed before `... on Source`?

Reply:
Good catch! Added the `__typename` to the `Error` object because it’s only relevant for narrowing down the error type; successful responses are just `Term`.